examples/options/src/track_propagation.cpp

```diff
@@ -52,11 +52,45 @@ track_propagation::track_propagation()
         "search-window",
         po::value(&m_search_window)->default_value(m_search_window),
         "Size of the grid surface search window");
-    m_desc.add_options()("rk-tolerance-mm [mm]",
+    m_desc.add_options()("rk-tolerance-mm",
                          po::value(&(m_config.stepping.rk_error_tol))
                              ->default_value(m_config.stepping.rk_error_tol /
                                              traccc::unit<float>::mm),
-                         "The Runge-Kutta stepper tolerance");
+                         "The Runge-Kutta stepper tolerance [mm]");
+    m_desc.add_options()("min-step-size-mm",
+                         po::value(&(m_config.stepping.min_stepsize))
+                             ->default_value(m_config.stepping.min_stepsize /
+                                             traccc::unit<float>::mm),
+                         "The minimum step size [mm]");
+    m_desc.add_options()("stepping-path-limit",
+                         po::value(&(m_config.stepping.path_limit))
+                             ->default_value(m_config.stepping.path_limit /
+                                             traccc::unit<float>::m),
+                         "The maximum path length for the stepper [m]");
```

Comment:
Is there a reason for the top one to have the unit but the bottom one not?

Reply:
:thinking: The naming was pretty all over the place already, I didn't notice that my AI helper was doing this.

Since I personally don't like putting `-mm` into the option itself, I'll make all new options skip having units in their names.